Sources/Datadog/Datadog.swift

```diff
@@ -247,11 +247,15 @@ public class Datadog {
                 telemetry: telemetry
             )
 
+            core.registerFeature(named: RUMFeature.featureName, instance: rum)
+
             if let instrumentationConfiguration = rumConfiguration.instrumentation {
                 rumInstrumentation = RUMInstrumentation(
                     configuration: instrumentationConfiguration,
                     dateProvider: dateProvider
                 )
+
+                core.register(feature: rumInstrumentation)
```

Comment:
Totally fine for now 👍, although I think `rumInstrumentation` shouldn't be a separate feature in V2. It's rather a component of RUM feature.

Reply:
Agreed, this will change when we move to v2 configuration/scope. What about `URLSession` instrumentation? it is used by both Tracing and RUM.
Given how the instrumentation works with custom `URLSessionDelegate`, it could make sense to have a separate config.. WDYT?

Reply:
Yes, agree - `URLSessoinInstrumentation` needs separate config 👌. I think we may need to put its code inside API module so it can be shared between RUM and Tracing. It also won't be a "feature" if compared to Logging, RUM and Tracing, so we may need to find a good shape for it.